core/src/index/knowhere/knowhere/index/vector_index/FaissBaseIndex.cpp

```diff
@@ -70,6 +70,7 @@ FaissBaseIndex::SealImpl() {
     faiss::Index* index = index_.get();
     auto idx = dynamic_cast<faiss::IndexIVF*>(index);
     if (idx != nullptr) {
+        KNOWHERE_LOG_DEBUG << __FUNCTION__ << " IVF READONLY " << std::boolalpha << idx->is_readonly();
```

Comment:
Stop using __FUNCTION__, use another tag to replace it.

Reply:
change __function__ to "Test before to_readonly:"